# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# result message
float64 itemsWashed	# the number of items washed
